command.ts

```diff
@@ -148,7 +159,7 @@ const yargs = require('yargs')
   .wrap(97)
 
 Object.keys(spec.properties).forEach((propertyKey) => {
-  const property = spec.properties[propertyKey]
+  const property = spec.properties[propertyKey as keyof typeof spec.properties]
```

Comment:
Can this assertion be done on line 161? It should be possible to do something like:

```
Object.keys(spec.properties).forEach((propertyKey: keyof typeof spec.properties) => {
```

Reply:
I'm afraid this is the cleanest method. Neither `Object.keys()`, nor `.forEach()` accept a type parameter, and typing `propertyKey` in the function parameters as the narrow union of string literals (keyof typeof spec.properties) results in the following error:

```
Types of parameters 'propertyKey' and 'value' are incompatible.

Type 'string' is not assignable to type '"header" | "types" | "preMajor" | "commitUrlFormat" | "compareUrlFormat" | "issueUrlFormat" | "userUrlFormat" | "releaseCommitMessageFormat" | "issuePrefixes"'.
```

Reply:
Actually, I have to correct myself here. Since we use 'propertyKey' later on in the function, the best thing to do here is to assign it like so:
```
const propName = propertyKey as keyof typeof spec.properties;
...
```

I'll make that change on next commit.